### DISCUSSION TITLE
Mobile Version

### DISCUSSION THREAD

**Discussion by yigitkaya24** [SebLague/Digital-Logic-Sim#129] [Ideas]
Please build mobile version, i want to use this program with my phone. I couldn't find any application like that.

**Comment by firecerne** [comment]
Just building the game on phone is easy enough, but you also have to worry about the fact that it might not really be playable. There are so many little pins, little interactions that need precision that you can't really do with your fingers on a touchscreen. The actual work needed is pretty massive.

**Reply by jerrecode** [reply]
You could look at Logic Circuit Simulator Pro and Atomate for some inspiration.

It is possible when scrapping the whole limited space inside chips thing.

Maybe use relative spacing of chips in an infinite plane centered on [0, 0] based on the position of the first chip and  moving Pins to inside the chip.

But I guess the simulation UI setup and having the Inputs on the left and the Outputs on the right might be for Usability reasons.

**Comment by parshwa282011** [comment]
I just tried doing some work like having a keyboard pop up if it is phone but still it's really hard to type even with that
